Yoast/YoastSEO.js
# Rewrite async/await to a Promise

## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:

* We don't support async/await in our plugin toolchain yet.

## Test instructions

This PR can be tested by following these steps:

* Run `grunt release` in Yoast SEO with this branch.
* You should be able to build the JavaScript files.
